spec/unit/fonts_spec.rb

```diff
@@ -24,9 +24,9 @@
 
     it 'should copy fonts if they are not found' do
       expect(executor).to receive(:execute)
-        .with('Copying roboto.ttf', "cp #{config.fonts_path}/roboto.ttf #{ENV['HOME']}/Library/Fonts/roboto.ttf")
+        .with('Copying roboto.ttf', "cp '#{config.fonts_path}/roboto.ttf' '#{ENV['HOME']}/Library/Fonts/roboto.ttf'")
       expect(executor).to receive(:execute)
-        .with('Copying bignoodle.otf', "cp #{config.fonts_path}/bignoodle.otf #{ENV['HOME']}/Library/Fonts/bignoodle.otf")
+        .with('Copying bignoodle.otf', "cp '#{config.fonts_path}/bignoodle.otf' '#{ENV['HOME']}/Library/Fonts/bignoodle.otf'")
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.